scripts/timer.py

```diff
@@ -0,0 +1,73 @@
+import json
+import datetime
+import threading
+import commands
+import ConfigParser
+from iota_cache import IotaCache
+from tag_helper import TagHelper
+from tm_rpc_client import Tendermint
+
+
+cf = ConfigParser.ConfigParser()
+cf.read("conf")
+iota_addr = cf.get("iota", "addr")
+iota_seed = cf.get("iota", "seed")
+cache = IotaCache(iota_addr, iota_seed)
+
+tm_addr = cf.get("tendermint", "addr")
+tm = Tendermint(tm_addr)
+
+
+def interval_work():
+    print("...work work work...")
```

Comment:
I think next step, we need to replace this with log, and this log info is really confusing